src/Spotify_Features.py

```diff
@@ -0,0 +1,223 @@
+#   GSH object, scopes, logging, and other feature level management. The end user in the applications should only
+#   ever have to instantiate this and choose which features they wish to use. No additional involvement needed.
+#
+# The paradigm of feature implementations is that if it is complicated enough to need multiple functions/ helpers it
+#   goes into it's own file/ class where 'SpotifyFeatures' will pass all appropriate args, our GSH object, and logger.
+#   The individual feature should define it's own scope needs (currently 'FEATURE_SCOPES') and apply them themselves.
+#
+# CURRENT FEATURES -
+#
+#   PLAYLIST GENERATORS ══════════════════════════════════════════════════════════════════════════════════════════════
+#   Generate Artist Playlist                    - reference generate_artist_playlist_from_id() / ..._from_playlist()
+#   Generate Monthly Release Playlist           - reference generate_monthly_release()
+#   Generate 'Generic' Artist Release Playlist  - reference generate_release_playlist()
+#
+#   DATA BACKUP ══════════════════════════════════════════════════════════════════════════════════════════════════════
+#   Spotify Library Backup                      - reference Backup_Spotify_Data.py
+#   Log Playback To Databases                   - reference Log_Playback.py
+#
+#   PLAYBACK MODIFIERS ═══════════════════════════════════════════════════════════════════════════════════════════════
+#   Shuffle Playlist                            - reference Shuffle_Styles.py
+#   Skip Track                                  - reference skip_track()
+#
+#   PLAYLIST ALTERATIONS ═════════════════════════════════════════════════════════════════════════════════════════════
+#   Distribute Tracks To 'Collections'          - reference Misc_Features.py distribute_tracks_to_collections...()
+#   Organize Playlist By Release Date           - reference Misc_Features.py reorganize_playlist()
+#
+#   MISC FEATURES ════════════════════════════════════════════════════════════════════════════════════════════════════
+#   Get Current Playback State                  - reference get_playback_state()
+#   Sanity Checks                               - reference Sanity_Tests.py
+#   Weekly Listening Report                     - reference Weekly_Report.py
+#
+# ════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
+import logging as log
+from datetime import datetime, timedelta
+from typing import Union
+
+import General_Spotify_Helpers as gsh
+from decorators import *
+
+# FEATURES
+from Misc_Features import MiscFeatures
+from Backup_Spotify_Data import BackupSpotifyData
+from Log_Playback import LogPlayback
+from Shuffle_Styles import Shuffler, ShuffleType
+
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+DESCRIPTION: 
```

Comment:
Blank description - should probably be populated?

Reply:
Good catch, had updated in later PR's but didn't backfill it.